bundles/org.eclipse.orion.client.ui/web/orion/parameterCollectors.js

```diff
@@ -81,8 +81,8 @@ define(['i18n!orion/nls/messages', 'orion/webui/littlelib'],
 				this._activeElements.onClose = onClose;
 				var focusNode = fillFunction(this._activeElements.parameterArea, this._activeElements.dismissArea);
 				if (!focusNode) {
-					// no parameters were generated.  
-					return false;
+					// no required parameters
+					return true;
```

Comment:
Should you get this information programmatically rather than assuming there's no required params?